amr-wind/physics/SyntheticTurbulence.cpp

```diff
@@ -304,11 +304,10 @@ AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE void interp_perturb_vel(
 {
     const int nz = t_grid.box_dims[2];
     const int nynz = t_grid.box_dims[1] * t_grid.box_dims[2];
-    // clang-format off
     // Indices of the 2-D cell that contains the sampling point
-    int qidx[4]{wt.jl * nz + wt.kl, wt.jr * nz + wt.kl, wt.jr * nz + wt.kr,
-                wt.jl * nz + wt.kr};
-    // clang-format on
+    amrex::Array<int, 4> qidx = {
```

Comment:
Is there an extra set of brackets here or is that the right format?

Reply:
good catch, fixed!